struct.go

```diff
@@ -6,13 +6,13 @@ import (
 	"time"
 )
 
-// EurekaUrlSlugs is a map of resource names -> eureka URLs
+// EurekaUrlSlugs is a map of resource names→Eureka URLs.
```

Comment:
I'm not a fan of non ascii characters. I'm just worried about some old-school people with weird editors...